notebooks/book1/03/gauss_infer_2d.ipynb

```diff
@@ -0,0 +1,786 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        A = cholesky(sigma)
</pre><p>We should use meaningful and lower case name instead of just "A". This is part of pep8 naming convention,</p><p>Also please apply this suggestion in remaining part of code. make function name, variables names meaning ful.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>